packages/ipfs-http-client/package.json

```diff
@@ -45,15 +45,14 @@
     "test:chrome": "aegir test -t browser -t webworker -- --browsers ChromeHeadless",
     "test:firefox": "aegir test -t browser -t webworker -- --browsers FirefoxHeadless",
     "lint": "aegir lint",
-    "prepare": "aegir build --no-bundle",
```

Comment:
Why was this removed?

Reply:
we dont need it here, the prepare for types is just to help with PR with gh url and mono-repo doesnt allow for gh url dep install. Plus it slows down everything

Reply:
> mono-repo doesnt allow for gh url dep install

Good news!  Now you can do:

```console
$ npm install https://gitpkg.now.sh/ipfs/js-ipfs/packages/ipfs-core?some-branch-identifier
```

https://gitpkg.now.sh